vuepress/docs/next/docs/getting-started/ent-bundle.md

```diff
@@ -2,150 +2,166 @@
 sidebarDepth: 2
 ---
 
-# Bundle Management - ent bundle CLI
-The Entando Bundle CLI extends the functionality of **ent** with a modular bundle management system. Starting with Entando 7.1, `ent bundle` orchestrates the lifecycle of a project,  packaging it into convenient recognizable bundles that can be inserted into any Entando Application. This composable approach takes advantage of a single project descriptor and repository, along with centralized [API management](ent-api.md) and [DB and Keycloak services](ent-svc.md).
+# Bundle Management
 
-This document covers `ent bundle` operations and the steps required to create an Entando Bundle.
+The Entando Bundle CLI extends the functionality of **ent** through a dedicated bundle management system. Starting with Entando 7.1, `ent bundle` orchestrates the lifecycle of a project, packaging it into a convenient and recognizable docker-based bundle that can be inserted into any Entando Application. This composable approach takes advantage of a single project descriptor and repository, along with centralized [API management](ent-api.md) and [DB and Keycloak services](ent-svc.md).
+
+Common `ent bundle` operations and the steps required to create an Entando Bundle are discussed in the following sections:
 
 1. [Initialization](#initialization)
 2. [Build](#build)
 3. [Run](#run)
 4. [Package](#package)
 5. [Publish](#publish)
-6. [Deploy](#ecr-commands)
+6. [Deploy](#deploy)
+
+This document also describes the series of `ent ecr` commands that manage bundle interactions with the [Entando Component Repository](../../docs/compose/local-hub-overview.md) (ECR). These are applicable to both docker-based bundles and their git-based predecessors (< Entando 7.1). Finally, notable commands applicable to creating and deploying git-based bundles only are discussed.
```

Comment:
i like the structure as-is because it emphasizes docker-based bundles, which aligns with the predominant focus of the content. it also introduces a psychological element where the reader automatically recognizes that git-based bundles are legacy. i think broaching the ecr and then touching on how those commands also apply to git-based bundles is very effective.